lib/ts/recipe/session/api/implementation.ts

```diff
@@ -82,6 +84,9 @@ export default function getAPIInterface(): APIInterface {
             if (session !== undefined) {
                 await session.revokeSession(userContext);
             }
+
+            clearSessionFromCookie(options.config, options.res);
```

Comment:
is this needed?

Reply:
Not strictly necessary, but since a user has been complaining about `signOutPost` not revoking their session in some cases (that I couldn't replicate), I thought it made sense just to go ahead and always clear the cookies even if we couldn't make sense of them.